Automattic/abacus
# Add experiment creation exposure events

**This PR adds Exposure Events to Experiment Creation:**

<img width="750" alt="Screen Shot 2020-08-26 at 2 08 47 pm" src="https://user-images.githubusercontent.com/971886/91267304-ac074880-e7a5-11ea-9aa4-5e6901d9fb9f.png">

- Video: https://d.pr/v/Wu8qsX
- Internally in the form I use an array of `{ key, value }` objects to represent the properties. It needs to be an array in form state to work with the `<FieldArray>`.
- Added extra integration testing: form state conversion to outbound.
- Added extra integration testing: using a real form state example to test `ExperimentsApi.create`




Fixes #262 

## How has this been tested?






- Automated tests that cover added/changed functionality
- Manual testing with production data (locally)
- Manual testing with mock data (locally)
- Additional manual testing instructions (please specify):
   - Create experiment using production data
   - Verify the fields are correct in the database
